src/App.js

```diff
@@ -0,0 +1,21 @@
+import { Route, Routes } from 'react-router-dom';
```

Comment:
You successfully created a single page application (SPA) using react-router-dom to serve each page without reloading 😎